### ISSUE TITLE
Train large hourglass from scratch?

### ISSUE THREAD

**Issue by zzzxxxttt** [xingyizhou/CenterNet#152]
Hi, I have a small question about training the hourglass net, is it possible to train the large hourglass net from scratch and reach the same performance of using pretrained backbone?

**Comment by zzzxxxttt** [comment]
I tried to train an hourglass net for 140 epochs from scratch, the final performance is 36.2/38.4/40.4, which is still ~4% lower than using pretrained model, maybe more epochs is needed to train from scratch?

**Comment by zzzxxxttt** [comment]
Update: I trained hourglass for 200 epochs with batch size 48 and initial learning rate 1e-3, the final mAP is 38.9/41.0/43.8, now it's very close with using pretrained model.

**Comment by xingyizhou** [comment]
Thanks for the report! Our results for `python main.py ctdet --exp_id coco_hg --arch hourglass --batch_size 24 --master_batch 4 --lr 2.5e-4 --gpus 0,1,2,3,4 --num_epochs 100 --lr_step 90` is 38.9/ 40.6/ 43.0. We are currently investigating longer schedules. Batch size 48 should be aligned with learning rate 5e-4. We do have observed using a larger learning rate will hurt the performance for the hourglass model.

**Comment by zzzxxxttt** [comment]
Thanks for your advice! I will try smaller lr later. And I think using syncBN will be helpful for train from scratch, since the batch size on master card is relatively small.

**Comment by qiuhaining** [comment]
> Thanks for the report! Our results for `python main.py ctdet --exp_id coco_hg --arch hourglass --batch_size 24 --master_batch 4 --lr 2.5e-4 --gpus 0,1,2,3,4 --num_epochs 100 --lr_step 90` is 38.9/ 40.6/ 43.0. We are currently investigating longer schedules. Batch size 48 should be aligned with learning rate 5e-4. We do have observed using a larger learning rate will hurt the performance for the hourglass model.

I guess the larger learning rate may bring about the dead relu！
